lib/gateway/context.js

```diff
@@ -1,7 +1,13 @@
 const vm = require('vm');
-
-function EgContextBase () {}
-
+const uuid62 = require('uuid-base62');
+function EgContextBase () {
+  this.requestId = uuid62.v4();
```

Comment:
Any linter advice on `requestId` vs `requestID`.  I tend to follow the latter pattern as that's what built-in JavaScript functions adhere to (e.g., `encodeURI`).

Reply:
ok, ObjectID, MySQL, GraphQL kind of thing.